flink-table/flink-table-api-scala/src/main/scala/org/apache/flink/table/api/expressionDsl.scala

```diff
@@ -33,7 +33,7 @@ import java.sql.{Date, Time, Timestamp}
 import java.time.{LocalDate, LocalDateTime, LocalTime}
 import java.util.{List => JList, Map => JMap}
```

Comment:
nit: Shall we move `ImplicitExpressionConversions` and `ImplicitExpressionOperations` into dedicated files? Personally, I don't like `expressionDsl.scala` much. `TableConversions` and `DataStreamConvertions` are also dedicated files.